pkg/operation/seed/seed.go

```diff
@@ -685,9 +694,32 @@ func BootstrapCluster(ctx context.Context, k8sGardenClient, k8sSeedClient kubern
 		}
 	}
 
+	if managedIngress(seed) {
+		seedClient := k8sSeedClient.Client()
+
+		// TODO(beckermax): Remove after next release - temporary migration logic.
```

Comment:
Hm, why? We need this always, right? We would even need the inverted migration when somebody disabled the managed ingress controller again.

Reply:
That is true :(. And yes we need the inverse as well. Feels wrong to attempt to set the ingressClass for all the shoot Ingresses on every Seed reconcile. Most of the time it will be a waste. But I have no better idea.